lib/src/list_notifier.dart

```diff
@@ -5,12 +5,23 @@ import 'package:flutter/foundation.dart' show ChangeNotifier, ValueListenable;
 import 'package:functional_listener/functional_listener.dart';
 
 /// A List that behaves like `ValueNotifier` if its data changes.
-/// it does not compare the elements on bulk operations
-/// If you set [notifyIfEqual] to `false` it will compare if a value passed value
-///  passed is equal to the existing value.
-/// like `list[5]=4` if the content at index 4 is equal to 4 and only call
+/// It does not compare the elements on bulk operations
```

Comment:
Actually this isn't longer the case. for most bulk operations it will compare elements.

Reply:
The only exceptions are 
![grafik](https://github.com/escamoteur/listenable_collections/assets/16635729/86c530a3-a7bb-4c47-9acc-39e16b6cae11)
perhaps we should list this functionnames there.  Most probably these functions will always change the List. 
I could not move the comments before the method name because then the inherited API doc of List won't appear when you hover over a method